squeal-postgresql/src/Squeal/PostgreSQL/Pool.hs

```diff
@@ -89,7 +89,7 @@ createConnectionPool
   -- ^ Maximum number of connections to keep open per stripe. The smallest acceptable value is 1.
   -- Requests for connections will block if this limit is reached on a single stripe, even if other stripes have idle connections available.
   -> io (Pool (K Connection schemas))
-createConnectionPool conninfo stripes idle maxResrc = liftBase $
+createConnectionPool conninfo stripes idle maxResrc = liftIO $
```

Comment:
since `createPool` from `unliftio-pool` is already lifted I don't think you should need a `liftIO` here.

Reply:
Thanks for the review! I will rebase on latest master and resolve all conflicts tomorrow